admin/partials/nginx-helper-general-options.php

```diff
@@ -548,6 +549,38 @@
 						</td>
 					</tr>
 				</table>
+                <table class="form-table rtnginx-table">
+                    <tr valign="top">
+                        <th scope="row">
+                            <h4>
+								<?php esc_html_e( 'Purge AMP URL:', 'nginx-helper' ); ?>
+                            </h4>
+                        </th>
+                        <td>
+                            <fieldset>
+                                <legend class="screen-reader-text">
+									<span>
+										&nbsp;
+										<?php
+										esc_html_e( 'purge amp urls', 'nginx-helper' );
+										?>
+									</span>
+                                </legend>
+                                <label for="purge_amp_urls">
+                                    <input type="checkbox" value="1" id="purge_amp_urls" name="purge_amp_urls" <?php checked( $nginx_helper_settings['purge_amp_urls'], 1 ); ?> />
```

Comment:
Why is this option enabled by default?

Reply:
@SH4LIN I have enabled it in accordance with the default behaviour since, by default, all other options—aside from comments—are enabled when purge is enabled.